src/view/edit-point.js

```diff
@@ -37,6 +37,8 @@ const getEditPointTemplate = (waypoint, i) => {
               </div>
               `);
   }).join('');
+  let buttonDeletion = isDeleting ? 'Deleting...' : 'Delete';
+  buttonDeletion = formType ? 'cancel' : buttonDeletion;
   return `<li class="trip-events__item"> 
```

Comment:
название переенной лучше на deleteButtonText поменяй, ну и Cancel с большой буквы сделай
```suggestion
  let deleteButtonText = isDeleting ? 'Deleting...' : 'Delete';
  deleteButtonText = formType ? 'Сancel' : deleteButtonText;
  return `<li class="trip-events__item"> 
```